PureBasicIDE/FindWindow.pb

```diff
@@ -6,6 +6,7 @@
 
 
 Procedure OpenFindWindow()
+  Protected FindFromSelection.b
```

Comment:
The `.b` should be removed. Always use the integer type in normal variables even if they just store a boolean value. Using a smaller type just wastes CPU cycles converting to/from the smaller data type on every access and trying to optimize memory usage on a single variable is a waste of effort. The smaller data types should only be used in data structures.

Reply:
> The `.b` should be removed. Always use the integer type in normal variables even if they just store a boolean value. Using a smaller type just wastes CPU cycles converting to/from the smaller data type on every access and trying to optimize memory usage on a single variable is a waste of effort. The smaller data types should only be used in data structures.

Thanks for letting me know, I didn't know. I will change it and also in other projects.
It would be good to have this, in the documentation.